### PR TITLE
Relative In-File Config Paths

### PR DESCRIPTION
**Pull request by ArquintL**
Changes behavior of In-File IncludeDir paths to be relative to the file in which they are instead of being relative to the location at which Gobra is executed.

### CODE REVIEW

File: src/test/scala/viper/gobra/BenchmarkTests.scala [hunk 1]
```diff
@@ -47,14 +47,17 @@ trait BenchmarkTests extends StatisticalTestSuite {
 }
 
 trait GobraFrontendForTesting extends Frontend {
+  val z3PropertyName = "GOBRATESTS_Z3_EXE"
+  val z3Exe: Option[String] = Option(System.getProperty(z3PropertyName))
```
Comment (Felalolf) [inline src/test/scala/viper/gobra/BenchmarkTests.scala:51]: What is this?

**Review by Felalolf — COMMENTED** [review]
I also checked it on Windows. I have one question, otherwise everthing looks fine.

File: src/test/scala/viper/gobra/BenchmarkTests.scala (hunk 1 above)
Comment (ArquintL) [reply src/test/scala/viper/gobra/BenchmarkTests.scala:51]: This reads the `GOBRATESTS_Z3_EXE` property passed with `-DGOBRATESTS_Z3_EXE`

**Review by Felalolf — APPROVED** [review]

**Comment by ArquintL** [comment]
I'll wait with merging this branch until @Felalolf branch on interfaces is merged as there are a few more in file configs to be adapted

**Review by Felalolf — APPROVED** [review]

**Pull request merged**